core/src/main/scala/kafka/log/Log.scala

```diff
@@ -17,32 +17,31 @@
 
 package kafka.log
 
+import java.io.{File, IOException}
+import java.text.NumberFormat
+import java.util.concurrent.atomic._
+import java.util.concurrent.{ConcurrentNavigableMap, ConcurrentSkipListMap}
+
+import com.yammer.metrics.core.Gauge
 import kafka.api.KAFKA_0_10_0_IV0
-import kafka.api.OffsetRequest
-import kafka.idempotence.IdMapping
-import kafka.utils._
 import kafka.common._
+import kafka.idempotence.ProducerIdMapping
```

Comment:
Not a fan of this package name.. Would `kafka.log` be a better place for the PID mapping another related code? They are really part of the log going forward.